models/issue_label.go

```diff
@@ -204,12 +237,93 @@ func NewLabels(labels ...*Label) error {
 	return sess.Commit()
 }
 
+// UpdateLabel updates label information.
+func UpdateLabel(l *Label) error {
+	if !LabelColorPattern.MatchString(l.Color) {
+		return fmt.Errorf("bad color code: %s", l.Color)
+	}
+	return updateLabel(x, l)
+}
+
+// DeleteLabel delete a label
+func DeleteLabel(id, labelID int64) error {
```

Comment:
I think you dont need `id` at all it's just iretating ...

Reply:
no you do, see above comments.

Reply:
ok - this comment is resolved